src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.cs

```diff
@@ -15,6 +15,8 @@ namespace System.IO.Compression.Tests
 {
     public partial class ZipFileTest_ConvenienceMethods : ZipFileTestBase
     {
+        public static bool IsNotMacOsHighSierraOrHigher => PlatformDetection.IsNotMacOsHighSierraOrHigher;
```

Comment:
Ah, yeah, that makes sense. The type name `ConditionalFact` was only added 11 days ago: https://github.com/dotnet/buildtools/pull/1634 and rel/2.0.0 doesn't have that change.